openmetadata-service/src/main/java/org/openmetadata/service/monitoring/EventMonitorFactory.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.openmetadata.service.monitoring;
+
+import com.google.common.annotations.VisibleForTesting;
+import lombok.Getter;
+import org.openmetadata.schema.monitoring.EventMonitorProvider;
+
+public class EventMonitorFactory {
+
+  @Getter private static EventMonitor eventMonitor;
```

Comment:
I don't think we need a static object on the EventMonitorFactory if this is going to be called only once at the start of the application.

Reply:
thanks for pointing this out. makes sense